cli/EAP7-1100-Add_the_ability_for_the_CLI_SSL_security_commands_to_be_able_to_obtain_a_server_certificate_from_Lets_Encrypt.adoc

```diff
@@ -0,0 +1,120 @@
+* The parameter '--lets-encrypt' should be used in combination with the '--interactive' parameter
+* After '--lets-encrypt parameter' is used user is asked for several information needed to obtain the certificate:
+** Key-store file name
+** Password
+** Account Key-store file name
+** Certificate account name
+** Contact url(s)
+** Domain name(s)
+** Alias
+** Enable SSL Mutual Authentication
+** Do you agree to Let's Encrypt terms of service?
+* Introduce '--ca-account=<name>' parameter to reuse existing --ca-account
+
+==== Example of the certificate interactive setup:
+[source,bash]
+----
+[standalone@localhost:9990 /] security enable-ssl-management --interactive --lets-encrypt
+Please provide required pieces of information to enable SSL:
+
+Key-store file name (default management.keystore): a
+Password (blank generated):
+Account Key-store file name (default accounts.keystore.jks):
+Certificate account name (default CertAuthorityAccount):
+Your domain name (must be accessible by the Let's Encrypt server at 80 & 443 ports) [example.com]: da14549f.ngrok.io
+Your contact email [admin@admin.com]: admin@da14549f.ngrok.io
+Alias (blank generated):
+Do you agree to Let's Encrypt terms of service? y/n (blank n): y
+Enable SSL Mutual Authentication y/n (blank n):
+
+SSL options:
+Certificate will be obtained from Let's Encrypt server and will be valid for 90 days
+account key store file: accounts.keystore.jks
+certification authority account name: CertAuthorityAccount
+key store file: a
+domain name: da14549f.ngrok.io
+password: GwA82e2S
+alias: alias-42723f73-ec17-4c84-9c20-160180490cf8
+Server keystore file a, certificate file a.pem and a.csr file will be generated in server configuration
+directory.
+
+Do you confirm y/n :y
+Subject    - CN=da14549f.ngrok.io
+Issuer     - CN=Let's Encrypt Authority X3, O=Let's Encrypt, C=US
+Valid From - Thu Nov 08 12:36:16 CET 2018
+Valid To   - Wed Feb 06 12:36:16 CET 2019
+MD5 : 83:e0:41:16:5e:f1:5b:b8:b3:4a:6f:94:5e:36:cd:03
+SHA1 : a2:98:38:82:9e:79:2c:11:3c:d4:2c:76:28:3e:6d:16:1c:7c:6f:25
+
+Subject    - CN=Let's Encrypt Authority X3,O=Let's Encrypt,C=US
+Issuer     - CN=DST Root CA X3, O=Digital Signature Trust Co.
+Valid From - Thu Mar 17 17:40:46 CET 2016
+Valid To   - Wed Mar 17 17:40:46 CET 2021
+MD5 : b1:54:09:27:4f:54:ad:8f:02:3d:3b:85:a5:ec:ec:5d
+SHA1 : e6:a3:b4:5b:06:2d:50:9b:33:82:28:2d:19:6e:fe:97:d5:95:6c:cb
+
+
+Accept certificate? [N]o, [T]emporarily, [P]ermanently : t
+Server reloaded.
+SSL enabled for http-interface
+ssl-context is ssl-context-7129ee02-add4-4acd-a39a-103a8c1ba495
+key-manager is key-manager-7129ee02-add4-4acd-a39a-103a8c1ba495
+key-store   is key-store-7129ee02-add4-4acd-a39a-103a8c1ba495
+----
+
+=== Nice-to-Have Requirements
+* Possibility to reuse key-store file, account key store file, Certificate account name
+
+=== Non-Requirements
+* Possibility to renew expired certificate
+* Possibility to revoke certificate
+
+== Test Plan
```

Comment:
It would be good to add a "Community Documentation" section to mention that the [documentation](https://github.com/wildfly/wildfly/blob/master/docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc#configure-ssltls) on the security commands will be updated to mention the new ```--lets-encrypt``` parameter.